### PR TITLE
feat: LT-2536-audit-organisation

### PR DESCRIPTION
**Pull request by rorysproule1**

### CODE REVIEW

File: ui_automation_tests/features/organisation.feature [hunk 1]
```diff
@@ -28,8 +31,10 @@ Feature: I want to add a company to LITE
     And I add a new HMRC organisation
     And I go to organisations
     Then HMRC organisation is registered
+    When I click the organisation
+    Then the "created" organisation appears in the audit trail
 
-  @LT_1086_adding_a_flag_to_an_organisation @regression
+  @LT_1086_adding_a_flag_to_an_organisation @regression @rory
```
Comment (phlptt) [inline ui_automation_tests/features/organisation.feature:37]: @ rory

Comment (rorysproule1) [reply ui_automation_tests/features/organisation.feature]: Already removed ;)

File: ui_automation_tests/pages/organisations_page.py [hunk 2]
```diff
@@ -29,3 +30,6 @@ def go_to_in_review_tab(self):
 
     def go_to_active_tab(self):
         self.driver.find_element_by_id(self.ACTIVE_TAB_ID).click()
+
+    def get_audit_trail_text(self):
```
Comment (phlptt) [inline ui_automation_tests/pages/organisations_page.py:34]: I'd have thought this function exists somwehere else

Comment (rorysproule1) [reply ui_automation_tests/pages/organisations_page.py]: Moved to shared :)

**Pull request merged**